packages/auth/lib/AccessToken.js

```diff
@@ -24,6 +23,12 @@ const scopeConfigs = {
   CLAIM_CARD: {
     exposeFields: ['email', 'cards'],
     ttlDays: 90
+  },
+  AUTHORIZE_SESSION: {
+    rolesIssuer: ['admin', 'supporter'],
```

Comment:
not happy with the naming here. Suggestion: `requiredRolesToIssue`

Reply:
Renamed to `requiredRolesToIssue` indeed.